spec/requests/ingredients_spec.rb

```diff
@@ -0,0 +1,11 @@
+require 'rails_helper'
+
+RSpec.describe "Ingredients", type: :request do
+  describe "GET /index" do
+    it "returns http success" do
+      get "/ingredients/index"
+      expect(response).to have_http_status(:success)
```

Comment:
seguramente falla por esto
```suggestion
      get "/ingredients/"
      expect(response).to have_http_status(:success)
```

Reply:
Creo que falla porque no hay rutas ni controladores de ingredientes 👀, lo mismo con todos los _specs_ que están en `spec/requests/`, yo los borraría si aun no están implementados.

Reply:
Perfecto. Los borramos por ahora. Gracias!